README.md

```diff
@@ -1,70 +1,234 @@
+## Live Demo <a name="live-demo"></a>
 
-### Advanced Configuration
+- [Live Demo](https://www.loom.com/share/df65b6d1292846c88af2370c5ea6f109)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Deployment
+<!-- GETTING STARTED -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## Getting Started <a name="getting-started"></a>
 
-### `npm run build` fails to minify
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Setup
+
+Clone this repository to your desired folder: 
+
+  1 - Go to the <> Code section in the repository. <br></br>
+  2 - Select the https tab and copy the url provided. <br></br>
+  3 - Use the git clone command in the root of the folder where you will save the project.<br></br>
+
+Note: For more information you can visit <a href="https://docs.github.com/es/get-started/getting-started-with-git/about-remote-repositories" target="_blank">About remote Repositories</a> and <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop" target="_blank">Adding and cloning repositories</a>.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code
+- Git
+- Npm
+- Node
+- React
+- Jest
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Amongisde the prerequisites, you will need to install the following dependencies: 
+- react-redux
+- redux toolkit
+- jest
+- babel
+- prop-types
+- react-dom
+- react-icons
+- react-router and react-router-dom
+- uuid
+
+All of these dependencies can be installed with [npm](https://www.npmjs.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command in your terminal (inside the project folder):
+
+- npm init
+- npm install
+- npm start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run Tests <a name="run-tests"></a>
+
+To run the the tests, you will have to run the following command in the console (inside the project folder): 
+
+- npm install
+
+And then you will have to run the following command in the console (inside the project folder): 
+
+- npm test
+
+You will see the outcome in the console
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Deployment <a name="deployment"></a>
+
+To deploy this project in a local enviroment, you will have to run:
```

Comment:
- [ ] It is required as part of project requirement to deploy your capstone project and add the link inside your README file. That requirement is yet to be implemented. Kindly do that.